client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -353,7 +390,10 @@ class CalypsoifyIframe extends Component {
 	}
 }
 
-const mapStateToProps = ( state, { postId, postType, duplicatePostId } ) => {
+const mapStateToProps = (
+	state,
+	{ postId, postType, duplicatePostId }: { postId: number; postType: string; duplicatePostId: any }
```

Comment:
The type of this argument is the already-defined `Props`. I.e., the props passed from outside to the connected component.

Reply:
oooh. good catch. this is interesting. I'll update to reflect that.

`connect()` does some [fun advanced typing](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-redux/index.d.ts#L113) to produce the right external type, but `mapStateToProps` isn't covered by that.